src/config/src/yaml.rs

```diff
@@ -113,6 +113,21 @@ pub trait Element: Sized {
         None
     }
 
+    /// From some string (should be used for top level elements if you want to embed the code)
+    fn from_string<'a>(src: &'a str) -> Result<Self, ConfigError> {
```

Comment:
@ebkalderon One use case of `from_string` would be to load the config from an android asset, which one can only read as bytes. Therefore the lifetime is necessary.